tools/tck/src/main/scala/org/opencypher/tools/tck/FeatureFormatChecker.scala

```diff
@@ -113,6 +133,10 @@ class FeatureFormatChecker extends TCKCucumberTemplate {
 
   After(_ => stepValidator.checkRequiredSteps())
 
+  private def shouldVerifyStepDetails: Boolean = {
+    !lastSeenQuery.matches(call)
+  }
```

Comment:
I don't understand why scenarios with `CALL` in them are allowed to bypass validation.